### ISSUE TITLE
Search in all open buffers?

### ISSUE THREAD

**Issue by infokiller** [mileszs/ack.vim#189]
Is that possible to do? I know about `AckWindow` but IIUC it only works on visible buffers.

**Comment by ches** [comment]
Nope, this is not a feature that currently exists. Seems like a reasonable addition.
